### DISCUSSION TITLE
feature: avante.nvim

### DISCUSSION THREAD

**Discussion by leCoq42** [LazyVim/LazyVim#4456] [Extra Requests]
### Did you check the docs?

- [X] I have read all the LazyVim docs
- [X] This is not a request for a new or existing extra (see above)

### Is your feature request related to a problem? Please describe.

"avante.nvim is a Neovim plugin designed to emulate the behaviour of the [Cursor](https://www.cursor.com/) AI IDE. It provides users with AI-driven code suggestions and the ability to apply these recommendations directly to their source files with minimal effort."

### Describe the solution you'd like

- The default config for avante has some conflicting mappings with lazyvim's default keybindings. 
- Avante filetype has to be added to the markdown preview config.

### Describe alternatives you've considered

CopilotChat.nvim, but avante.nvim seems to have a lot more traction currently. Also avante.nvim allows users two choose their own llm provider.

**Comment by dpetka2001** [comment]
https://github.com/LazyVim/LazyVim/blob/main/.github/ISSUE_TEMPLATE/feature_request.yml

> IMPORTANT: ideas or requests related to extras, should go in the [Extra Requests](https://github.com/LazyVim/LazyVim/discussions/categories/extra-requests)
discussion forum.

**Comment by bmulholland** [comment]
Dupe of https://github.com/LazyVim/LazyVim/discussions/4402
